lib/install/management/delete.go

```diff
@@ -66,7 +100,7 @@ func (d *Dispatcher) DeleteVCHInstances(conf *metadata.VirtualContainerHostConfi
 		if name == conf.Name {
 			continue
 		}
-		if _, err = d.deleteVM(child); err != nil {
+		if _, err = d.deleteVM(child, d.force); err != nil {
```

Comment:
This is ignoring the extra files that will be in those containerVM directories - the containerVM folder will not be deleted.
Please move the folder deletion for VMs into the deleteVM function - I don't think there's any case at this time where we don't want to delete the folder as well - can you identify any?